### PR TITLE
Add parameter to copy custom init.d scripts.

### PR DESCRIPTION
**Pull request by sebastianludwig**
As discussed in #151 I added a parameter to copy custom init.d scripts to Jenkins.

### CODE REVIEW

**Comment by sebastianludwig** [comment]
@geerlingguy Not "All checks have failed", one test got [stuck installing Java](https://travis-ci.org/geerlingguy/ansible-role-jenkins/jobs/262703410#L545). The [added test succeeded](https://travis-ci.org/geerlingguy/ansible-role-jenkins/jobs/262703423#L798). Can you trigger Travis again?

**Comment by sebastianludwig** [comment]
Never mind, I forgot to update the README. This time all checks passed 🎉

**Comment by sebastianludwig** [comment]
@geerlingguy is anything missing I can do?

File: tests/test-initd-scripts.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+- hosts: all
+
+  vars:
+    jenkins_initd_files:
+      - "{{ role_path }}/tests/README.md"
```
Comment (geerlingguy) [inline tests/test-initd-scripts.yml]: Is it possible to add in any more meaningful / practical tests here? Something that would be an actual init script that would affect Jenkins, to make sure that if we copy one in, Jenkins actually boots and uses it and doesn't fail?

**Comment by sebastianludwig** [comment]
@geerlingguy sorry that it took so long but I finally got around to add a real script to the test case. Anything else you'd like to have changed?

**Comment by jgerry** [comment]
I tried this PR today as it's something I need - works great! Merge away. I also added another option like this to manually install `.hpi` files for private plugins. My job requires our private pre-configured Okta plugin which needed this.

**Comment by nickbroon** [comment]
This is definitely very useful, and I'd love to see it merged.
It's specifically for the example case of starting jenkins in quiet mode that I'd like to use it.
https://wiki.jenkins.io/display/JENKINS/Post-initialization+script documents this.
Can this be merged?

File: tests/test-initd-scripts.yml (hunk 1 above)
Comment (nickbroon) [reply tests/test-initd-scripts.yml]: I think this outstanding review change request has been made?

**Pull request closed without merging**